src/main/java/hudson/plugins/templateproject/ProxySCM.java

```diff
@@ -106,7 +102,7 @@ public boolean pollChanges(AbstractProject project, Launcher launcher,
 		return getProjectScm().pollChanges(project, launcher, workspace, listener);
 	}
 
-	@Extension
+	@Extension(optional = true)
```

Comment:
What do you expect to happen here?

* SCMDescriptor is in the core, it always exists
* There is no static initializer which would break classloading if a class is missing

Reply:
The class is using `org.jenkinsci.plugins.multiplescms.MultiSCMRevisionState` from Multiple SCMs plugin. But `multiple-scms` plugin is an optional dependency of Template project plugin, so we have 

Would the second part of the patch be enough? I didn't think so.

Reply:
OK, nice that it works as is. I assumed that it may ignore the parent class initialization in this case

Reply:
I tested with an empty Jenkins. I could have break more than I wanted. So now I'm scared.